docs/tutorials/99_other/publish.md

```diff
@@ -0,0 +1,54 @@
+# Tutorial - How to Publish data and metadata on the OEP
+
+<!-- keep img below title and without align="left"  -->
+<img src="https://raw.githubusercontent.com/OpenEnergyPlatform/academy/develop/docs/data/img/OEP_logo_2_no_text.svg" alt="OpenEnergy Platform" height="75" width="75" />
+
+## Learnings
+
+This tutorials will enable you to:
+
+- Publish your data and metadata 
+	- with an embargo
+	- to a topic
+
+## Requirements
+
+- OEP Account
+- a table or dataset with metadata uploaded to the [OpenEnergyPlatform](https://openenergyplatform.org/) (OEP)
+
+
+## Introduction
+
+This course will teach you step-by-step how to publish your data and metadata. Comment: Later will be a corresponding course linked here.
```

Comment:
Here it would be good to explain what publication means in the context of the OEP. Also what embargo means. But this is not directly the content of the tutorial, I think ... well, I don't want to complicate things too much, so we can just put this information here.

Publish:
- Move table to a target data theme
- The data must have an Open Data licence -> SPDX-Datalicense list
- As long as the data is published, it cannot be changed. 
- It is possible to "unpublish" data .

Embargo:
- A period of time during which data cannot be accessed without permission. 
- The OEP Dataviewer is blocked, API-based data access is blocked.
- The embargo can be set during data upload and data publication.
- Only embargo periods of 6 months or 1 year are possible.
- Embargoed data still requires an open license in order to be published.

Reply:
This looks to me like it could become part of the [04 - Upload - Contribute data to the OEP](https://openenergyplatform.github.io/academy/courses/04_upload/) course. What do you think @han-f ?